data.tf

```diff
@@ -0,0 +1,8 @@
+data "github_organization" "this" {
+  name = var.owner
+}
```

Comment:
It was added in the beginning on prototyping. I hoped it can be useful with other resources.
But currently it is not used. It is safe to be remove. Should I?

Reply:
removed